src/main/scala/com/fulcrumgenomics/cmdline/FgBioTool.scala

```diff
@@ -41,4 +41,7 @@ trait FgBioTool {
 
   /** Generates a new validation exception with the given message. */
   def invalid(message: String) = throw new ValidationException(message)
+
+  /** Generates a validation exception if the test value is false. */
+  def validate(test: Boolean, message: => String) = if (!test) throw new ValidationException(message)
```

Comment:
I'm not sure I see the point.  `invalid` always evaluates it's parameter, where as `validate` only does if the `test` fails.